modules/android/partials/release-notes/couchbase-mobile-android-release-note.3.2.0-beta.adoc

```diff
@@ -0,0 +1,202 @@
+[#maint-3-2-0-beta-1]
+== 3.2.0 Beta 1 -- March 2024
```

Comment:
Does this still need to say "Beta 1"?
It looks like that essentially *was* the beta for Android? e.g. the beta versions just specify which mobile SDKs are involved?